packages/btcindexer/db/migrations/0004_add_deposit_senders.sql

```diff
@@ -0,0 +1,8 @@
+-- This table links a Bitcoin transaction ID to its sender addresses.
+CREATE TABLE nbtc_deposit_senders (
+    tx_id TEXT NOT NULL,
+    sender TEXT NOT NULL,
+    PRIMARY KEY (tx_id, sender)
+);
+
+CREATE INDEX idx_sender ON nbtc_deposit_senders (sender);
```

Comment:
change the order of the primary key to be `sender, tx_id` and then we don't need this extra index

Reply:
done